tests/test_mgxs_library_nuclides/test_mgxs_library_nuclides.py

```diff
@@ -25,8 +25,12 @@ def _build_inputs(self):
         # Initialize MGXS Library for a few cross section types
         self.mgxs_lib = openmc.mgxs.Library(self._input_set.geometry)
         self.mgxs_lib.by_nuclide = True
-        self.mgxs_lib.mgxs_types = ['transport', 'nu-fission',
-                                    'nu-scatter matrix', 'chi']
+        self.mgxs_lib.mgxs_types = ['total', 'transport', 'nu-transport',
```

Comment:
There was. I didnt update the inputs after removing the summary=True option
:-(

On Sun, May 29, 2016 at 2:28 PM, Will Boyd <notifications@github.com> wrote:

> In tests/test_mgxs_library_nuclides/test_mgxs_library_nuclides.py
> <https://github.com/mit-crpg/openmc/pull/661#discussion_r65009348>:
>
> > @@ -25,8 +25,12 @@ def _build_inputs(self):
> >          # Initialize MGXS Library for a few cross section types
> >          self.mgxs_lib = openmc.mgxs.Library(self._input_set.geometry)
> >          self.mgxs_lib.by_nuclide = True
> > -        self.mgxs_lib.mgxs_types = ['transport', 'nu-fission',
> > -                                    'nu-scatter matrix', 'chi']
> > +        self.mgxs_lib.mgxs_types = ['total', 'transport', 'nu-transport',
>
> Also, looks like none of your commits have passed Travis yet, so there may
> still be a lingering issue. Make sure your version of Pandas is consistent
> with that used on the Travis image, that is always a hang up.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/661/files/117ca2e1b24342835fb85b56d6b4afe5bf03ef4f#r65009348>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA_TMwZVFdxpRbV-E0z7YBJy9inxGJtvks5qGdrpgaJpZM4IpTfD>
> .
>